graphics/managed_surface.cpp

```diff
@@ -728,6 +729,77 @@ void ManagedSurface::transBlitFromInner(const Surface &src, const Common::Rect &
 
 #undef HANDLE_BLIT
 
+Common::Rect ManagedSurface::blendBlitFrom(const ManagedSurface &src, const Common::Rect &srcRect,
+										   const Common::Rect &destRect, int flipping,
+										   const uint32 colorMod,
+										   const TSpriteBlendMode blend,
+										   const AlphaType alphaType) {
+	Common::Rect srcArea = srcRect, dstArea = destRect;
+	if (format != getSupportedBlendBlitPixelFormat() || src.format != getSupportedBlendBlitPixelFormat()) {
```

Comment:
For extensibility, I would flip it around. e.g. ```C++
if (!formatIsSupportedByBlendBlit(format) || !formatIsSupportedByBlendBlit(src.format))
```
So, once more formats are supported, we then do not need to return here